hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/rollback/RollbackUtils.java

```diff
@@ -122,6 +122,7 @@ static HoodieRollbackStat mergeRollbackStat(HoodieRollbackStat stat1, HoodieRoll
       List<ListingBasedRollbackRequest> partitionRollbackRequests = new ArrayList<>();
       switch (instantToRollback.getAction()) {
         case HoodieTimeline.COMMIT_ACTION:
+        case HoodieTimeline.REPLACE_COMMIT_ACTION:
```

Comment:
Yes, this was missed because MOR is not well tested with clustering/insert_overwrite. New test covers this. Btw, There may be other places where we might be missing similar handling of REPLACE_COMMIT. I tried doing an audit, but its very tedious. So, I may have missed some places. Let me know if you find any.